rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -679,14 +684,14 @@ def rotkehlchen_instance(
 
 @pytest.fixture
 def rotkehlchen_api_server_with_exchanges(
-        rotkehlchen_api_server,
-        added_exchanges,
-        gemini_test_base_uri,
-        gemini_sandbox_api_secret,
-        gemini_sandbox_api_key,
```

Comment:
Typed this function because I had issues with this fixture until I figured what was wrong